Services/user_service.py

```diff
@@ -0,0 +1,17 @@
+from Models.coordinates import Coordinates
+from Models.user_data import UserData
+
+
+class UserService:
+    def __init__(self, args):
+        self.args = args
+
+    def get_user_data(self):
+        self.validate_user_args()
+        user_coordinates = Coordinates(self.args[0], self.args[1])
+        user_url = self.args[2]
+        return UserData(user_coordinates, user_url)
+
+    def validate_user_args(self):
+        if len(self.args) < 3:
+            raise IndexError("Error - must provide 3 arguments as input: latitude logitude csv_url")
```

Comment:
Really good that you are catching different errors and let the user know if something went wrong, but how could we do to have all the errors in one place and just use them from there?

Reply:
I guess we can have a module to handle just validation